content/guides/tina-cloud/starter/structure.md

```diff
@@ -76,6 +76,8 @@ The top-level key in the schema is an array of _collections_, a `collection` inf
 
 Templates are responsible for defining the shape of your content, you'll see in the schema for this starter that we use `templates` for `collections` as well as `blocks`. If you look at the `landingPage` template, you'll notice that it has a set of `blocks`, which are also templates.
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/EwewKEHHkd4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Can you change the iframe to allowfullscreen=true

Reply:
Good catch!